config/locales/models/user.en.yml

```diff
@@ -95,6 +98,28 @@ en:
           <p class="govuk-body">
             Doing so will allow them to sign in to the application.
           </p>
+      anonymise:
+        title: Anonymise user?
+        content_html: |
+          <p class="govuk-body">
+            You are about to anonymise the user %{email}.
+          </p>
+
+          <p class="govuk-body">
+            Doing so will prevent the user from signing in to the application.
+          </p>
```

Comment:
I suspect this para is redundant; if we're anonymising, we are by definition doing so from the Deactivated tab and consequently they're already unable to sign in?